docker-compose-files/compose.sh

```diff
@@ -157,7 +157,7 @@ while [[ $# -gt 0 ]]; do
       # RUN ONLY setup containers
       PROFILE_ARG="--profile setup-influx-test-db"
       POST_ACTION="influx_setup_finish"
-      DETACH_ARG="-d"
```

Comment:
Talked with Sailaja.  The DETACH_ARG is modified for debug.  This should be not changed long term.

Reply:
The -d option helps with shutting everything down after the automatic setup.